grpc/pom.xml

```diff
@@ -92,6 +92,12 @@
       <version>2.0.1</version>
       <scope>compile</scope>
     </dependency>
+      <dependency>
```

Comment:
I've added this dependency because we need to transform UUID to `bytes[]`, and this logic is already implemented in the `UUIDGen.decompose`. However, the `UUIDGen` class is in this dependency. If that's problematic, we may decide to inline the decompose method. wdyt?

Reply:
I'd much rather inline it than pull in the cql module with all of it's dependencies.

Reply:
`UUIDGen` comes from C* OSS, right? I think we wanted to remove that dependency from the `cql` module too (or at least contain it). +1 to adding a local utility class for this in SG.

Reply:
Maybe use a `string` instead of `bytes`?  It might also be nicer for the user too, since it's a UUID.

Reply:
+1 on using `string`. I thought that the schema in:
```
message Response {
  Payload result_set = 1;
  bytes tracing_id = 2;
  repeated string warnings = 3;
}
```
cannot be changed in a non-compatible way. But if we can, That would solve this. 
Done in: https://github.com/stargate/stargate/pull/1062/commits/432a7954ddba51f670aa14ad45387bb0c9f4664f